etcdmgmt/etcd-mgmt.go

```diff
@@ -160,3 +147,115 @@ func isETCDStartNeeded() (bool, int) {
 	}
 	return start, pid
 }
+
```

Comment:
Every function at the beginning should have a comment what that function does. Please change accordingly for all new functions